src/core/public/doc_links/doc_links_service.ts

```diff
@@ -126,6 +127,8 @@ export class DocLinksService {
         addData: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/connect-to-elasticsearch.html`,
         kibana: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/index.html`,
         elasticsearch: {
+          dateMathIndexNames: `${ELASTICSEARCH_DOCS}date-math-index-names.html`,
```

Comment:
nit: Could alternatively live under `date` with the other dateMath link, but no strong feelings on this one way or the other.

Reply:
Thanks, I'll make that change!